x/examples/outline-pwa/README.md

```diff
@@ -1,19 +1,340 @@
+
+    ```kotlin
+    import android.os.*
+    import mobileproxy.*
+    import androidx.webkit.*
+    ```
+ 
+  * Now, in your `MainActivity.kt`, confirm proxy override is available in `onCreate`:
+
+    ```kotlin
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        if (WebViewFeature.isFeatureSupported(WebViewFeature.PROXY_OVERRIDE)) {
+            // Proxy override is supported
+        }
+    }
+    ```
+
+  * Initialize `mobileproxy` with a smart dialer in `onCreate`. Don't forget to replace `www.your-pwa-url.com`!:
+
+    ```kotlin
+    private var proxy: Proxy? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        if (WebViewFeature.isFeatureSupported(WebViewFeature.PROXY_OVERRIDE)) {
+            this.proxy = Mobileproxy.runProxy(
+                "127.0.0.1:0",
+                Mobileproxy.newSmartStreamDialer(
+                    Mobileproxy.newListFromLines("www.your-pwa-url.com"),
+                    "{\"dns\":[{\"https\":{\"name\":\"9.9.9.9\"}}],\"tls\":[\"\",\"split:1\",\"split:2\",\"tlsfrag:1\"]}",
+                    Mobileproxy.newStderrLogWriter()
+                )
+            )
+        }
+    }
+    ```
+
+  * Proxy all app requests after the proxy is initialized using `ProxyController`:
+
+    ```kotlin
+    // NOTE: This affects all requests in the application
+    ProxyController.getInstance()
+        .setProxyOverride(
+            ProxyConfig.Builder()
+                .addProxyRule(this.proxy!!.address())
+                .build(),
+            {
+                runOnUiThread {
+                    // Capacitor does not expose a way to defer the loading of the webview,
+                    // so we simply refresh the page
+                    this.bridge.webView.reload()
+                }
+            },
+            {}
+        )
+    ```
+
+  * Turn off the proxy in `onDestroy`:
+
+    ```kotlin
+    override fun onDestroy() {
+        this.proxy?.stop(3000)
+        this.proxy = null
+
+        super.onDestroy()
+    }
+    ```
+
+## Verify with Packet Tracing
+
+* **iOS:** Load XCode and connect your iOS device. Make sure that your device allows your developer certificate (Check `Settings > General > VPN & Device Management` on your iOS device). Go to `Product > Profile` in XCode and select the "Network" option from the list of presets. Pressing the record button on the window that pops up should launch your app and record its traffic. Once done, set the view on the packet trace data to `History` and among the results you should see the DNS address `dn9.quad9.net`. 
+* **Android:** Start the emulator with `npx cap run android`. Use Wireshark to capture network traffic. Filter by `ip.addr == 9.9.9.9` (your chosen DNS server). You should see TCP and TLS traffic, indicating that your app is using DNS over HTTPS (DoH).
+
```

Comment:
It would be nice to have a section here on how to change the icon and splash screen, since that may be Capacitor-specific, and clients will want that for sure.